configure_cake_wallet.sh

```diff
@@ -27,9 +27,41 @@ if [ "$PLATFORM" == "$ANDROID" ]; then
     cd scripts/android
 fi
 
+echo "============================ BUILDING DEPS ============================"
+
 source ./app_env.sh cakewallet
 ./app_config.sh
+./build_decred.sh
+
+if [ "$PLATFORM" == "$MACOS" ]; then
+   ./gen.sh
+fi
+
+echo "=================== generate secrets, localization ===================="
+
 cd ../.. && flutter pub get
-#flutter packages pub run tool/generate_localization.dart
+flutter packages pub run tool/generate_localization.dart # If this fails, add --force flag to the command
```

Comment:
This was commented out upstream. I'm not sure why. I'm not even sure what purpose this script is meant to serve, because for a first time run, you need to run the build scripts. So maybe the use case for this script doesn't require generating localization files?

In any case, I'm not very sure we should modify this file. It likely works for them as is, currently. We can instead only add scripts for macos and ios.